packages/cli/yaml/yaml-schema/src/schemas/HttpQueryParameterSchema.ts

```diff
@@ -4,7 +4,8 @@ import { extendTypeReferenceSchema } from "./TypeReferenceSchema";
 
 export const HttpQueryParameterSchema = extendTypeReferenceSchema(
     DeclarationWithNameSchema.extend({
-        "allow-multiple": z.optional(z.boolean())
+        "allow-multiple": z.optional(z.boolean()),
+        "query-parameter-representation": z.optional(z.enum(["EXPLODED", "COMMA_SEPARATED"]))
```

Comment:
- `encoding` or `format` are probably better keys than `query-parameter-representation` 
- lets not make the value screaming snake case (`exploded` and `comma_delimited`) 
- you can make the typing a union here where if `allow-multiple` is true then `encoding` needs to be specified (by leveraging z.union())

Reply:
Updated!